### PR TITLE
Add solvers to model checker json docs

### PR DESCRIPTION
**Pull request by leonardoalt**
Forgot in the initial PR

### CODE REVIEW

File: docs/using-the-compiler.rst [hunk 1]
```diff
@@ -410,6 +410,9 @@ Input Description
           "engine": "chc",
           // Choose whether to output all unproved targets. The default is `false`.
           "showUnproved": true,
+          // Choose which solvers should be used, if available.
+          // See the Formal Verification section for the solvers description.
+          "solvers": ["smtlib2", "z3"],
```
Comment (cameel) [inline docs/using-the-compiler.rst]: Maybe it would be better to list all available values?

**Review by cameel — DISMISSED** [review]
I'd list all available values but even without that it's good.

File: docs/using-the-compiler.rst (hunk 1 above)
Comment (leonardoalt) [reply docs/using-the-compiler.rst]: Done

**Review by Assawal — APPROVED** [review]
Thanks for the maintainability

**Review by cameel — APPROVED** [review]

**Pull request merged**